content/docs/fragments.md

```diff
@@ -0,0 +1,144 @@
+---
+id: fragments
+title: Fragments
+permalink: docs/fragments.html
+---
+
+A common pattern in React is for a component to return multiple elements. Fragments let you group a list of children without adding extra nodes to the DOM.
+
+Fragments look like empty JSX tags:
+
+```js
+render() {
+  return (
+    <>
```

Comment:
Since neither Create React App nor CodePen (two tools we suggest in docs) support this syntax it seems odd to me that we’ve chosen to use it here.

I would prefer that we use React.Fragment everywhere on this page except a small footnote about the new syntax until it’s mainstream.